### PR TITLE
Colormap event specifications

### PR DESCRIPTION
**Pull request by favreau**

### CODE REVIEW

File: doc/feature/colorMaps.md [hunk 1]
```diff
@@ -7,253 +7,102 @@ applications. An exemplary case is the transfer functions that map density to
...
 ## Requirements
 
-- Represent RGBA colormaps with an independent TF for each channel. The basic
+- Represent RGBA color maps with an independent TF for each channel. The basic
   TF definition should be reusable for the more generic case.
-- The C++ helper class will encapsulate a colormap and allow the conversion of
-  the network representation into a OpenGL ready lookup table for creating a
-  texture.
+- Represent extra material properties such as light emission, color contribution
+  and specular color.
```
Comment (dnachbaur) [inline doc/feature/colorMaps.md]: no more specular color

File: doc/feature/colorMaps.md [hunk 2]
```diff
@@ -7,253 +7,102 @@ applications. An exemplary case is the transfer functions that map density to
...
-using normalized coordinates for the x axis and then recalculate the positions
-when applied to an application.
+A simple event defines a range and a transfer function for each 'channel'. In
+our case, channels are the diffuse color, the opacity (alpha), the light
+emission, the specular color, and the contribution to the geometry (how much of
+the geometry material is impacted by the transfer function).
+Each lookup table can have a variable number of values.
 
-### Messages
+A color maps cannot be partially updated, the message always describes the
```
Comment (dnachbaur) [inline doc/feature/colorMaps.md]: As, not A?

**Review by dnachbaur — COMMENTED** [review]
+1

File: doc/feature/colorMaps.md (hunk 1 above)
Comment (favreau) [reply doc/feature/colorMaps.md]: Well, nobody was really convinced, so I removed it. I am OK to adding it back, no opinion there. We can always add it later if any of the renderers needs/wants it.

File: doc/feature/colorMaps.md (hunk 2 above)
Comment (favreau) [reply doc/feature/colorMaps.md]: Nope, A is fine.

**Comment by chevtche** [comment]
+2

**Review by dnachbaur — APPROVED** [review]

**Pull request merged**